.github/ISSUE_TEMPLATE/docs.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Docs
+about: Issue related to Salt Documentation
+title: "[DOCS]"
+labels: Documentation
+assignees: ''
+
+---
+
+**Description**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Type of documentation**
+This could be Salt documentation, Salt modules, the Salt Repo or the Getting Started guide.
+
+**Location or format of documentation**
+Insert page URL if applicable.
+
+**Suggested Fix**
+What did you expect to see in the documentation that is missing or needs updating?
+
```

Comment:
**Suggestion (non-blocking):** I wonder if we should move the Suggested Fix section right after the Description section. I could see how a person submitting an issue would logically think of that part next.